src/Microsoft.Azure.WebJobs/QueueAttribute.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.Azure.WebJobs
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1813:AvoidUnsealedAttributes")]
     [AttributeUsage(AttributeTargets.Parameter)]
     [DebuggerDisplay("{QueueName,nq}")]
-    public class QueueAttribute : Attribute
+    public class QueueAttribute : StorageAccountAttribute
```

Comment:
The "is a" relationship here kind of doesn't make sense. If we had a StorageAttribute (rather than StorageAccountAttribute) it does make more sense. QueueAttribute "is a" StorageAttribute. What if we just introduced this new StorageAccount base type and used that?